web/template/create_document.gohtml

```diff
@@ -38,12 +38,11 @@
 
             {{ else if .TemplateSelected.TemplateId }}
                 <a href="create-document?id={{ .Case.ID }}&case={{ .Case.CaseType }}" class="govuk-back-link">Back</a>
-
                 {{ template "case-details" . }}
                 {{ template "select-insert" . }}
 
             {{ else }}
-                {{ template "case-details" . }}
+                <p class="govuk-body govuk-!-margin-top-8"><strong>{{ if eq .Case.SubType "pfa" }}PFA{{ else if eq .Case.SubType "lpa" }}LPA{{ end }} {{ .Case.UID }}</strong></p>
```

Comment:
To avoid duplicating the `case-details` partial, you could just wrap in a `div` to add the margin:
```suggestion
                <div class="govuk-!-margin-top-8">{{ template "case-details" . }}</div>
```